rustls/src/msgs/persist.rs

```diff
@@ -107,22 +110,28 @@ impl Codec for ClientSessionValue {
             server_cert_chain,
         })
     }
+
+    pub fn get_encoding(&self) -> Vec<u8> {
+        let mut result = Vec::new();
+        self.encode(&mut result);
+        result
+    }
```

Comment:
This appears to be unused?

Reply:
It is used, which is why I added it. It has a default implementation in the `Codec` trait.

Reply:
If you have access to the codecov report, you should be able to see it is actually executed too.

Reply:
I'm very confused. Your custom implementation appears to be identical to the default implementation -- so why is it useful to add one here?

Reply:
> I'm very confused. Your custom implementation appears to be identical to the default implementation -- so why is it useful to add one here?

This type no longer implements the `Codec` trait because the `Codec` trait's `read` method doesn't allow passing in additional arguments. Since it no longer implements `Codec`, it can no longer use the default implementation of `Codec::get_encoding()`.

Reply:
Ah, sorry -- the change from `impl Codec for` to `impl ` was a bit subtle.